articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -242,6 +242,40 @@ image_object_detection_job = automl.image_object_detection(
 ```
 ---
 
+## Choose orchestration (preview)
+> [!IMPORTANT]
+> This feature is currently in public preview. This preview version is provided without a service-level agreement. Certain features might not be supported or might have constrained capabilities. For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).
+
+# [Azure CLI](#tab/cli)
+
+[!INCLUDE [cli v2](../../includes/machine-learning-cli-v2.md)]
+
+You can choose to use the legacy runtime or the new components (in preview) which run on [Azure Machine Learning pipelines](concept-ml-pipelines.md). You can indicate that you want to use the pipeline orchestration by setting `_automl_subgraph_orchestration` flag to `"true"` in the job properties. For example,
+
+```yaml
+properties:
+  _automl_subgraph_orchestration: "true"
+```
+
+
+# [Python SDK](#tab/python)
+
+[!INCLUDE [sdk v2](../../includes/machine-learning-sdk-v2.md)]
+
+You can choose to use the legacy runtime or the new components (in preview) which run on [Azure Machine Learning pipelines](concept-ml-pipelines.md). You can indicate that you want to use the pipeline orchestration by setting `_automl_subgraph_orchestration` flag to `"true"` in the job properties. For example,
```

Comment:
legacy runtime or the new components --> This might not be very clear to the users, what do we mean by legacy runtime.

Reply:
@bhimar - what's our thought on it?